Tests/SwiftDocCTests/Converter/DocumentationContextConverterTests.swift

```diff
@@ -26,6 +26,12 @@ class DocumentationContextConverterTests: XCTestCase {
         
         for identifier in context.knownPages {
             let documentationNode = try XCTUnwrap(try context.entity(with: identifier))
+
+            guard !context.onlyHasSnippetRelatedChildren(for: documentationNode.reference),
+                  documentationNode.kind != .snippet,
+                  documentationNode.kind != .snippetGroup else {
+                      continue
```

Comment:
Since `knownPages` already filter out pages with only snippet related content, would it make sense to fail the test if there's an unexpected snippet page?

Reply:
Snippets don’t get their own page but are in the topic graph (for linking), so the fake module page would be empty. That knownPages filter is to keep those empty module topics from getting registered as a top level page. So snippets are expected in this test.